index.js

```diff
@@ -31,6 +31,14 @@ module.exports = {
     return this.filterHelpers(tree, new RegExp(moduleRegexp, 'i'));
   },
 
+  treeForApp: function() {
+    // see: https://github.com/ember-cli/ember-cli/issues/4463
+    var tree = this._super.treeForApp.apply(this, arguments);
+    var moduleRegexp = /^helpers\//;
```

Comment:
Regex taken from `ember-composable-helpers`. The regex used in the `treeForAddon` can't be used here as the paths in the tree are different.